types/node-fetch/index.d.ts

```diff
@@ -204,7 +204,8 @@ export type BodyInit =
     | ArrayBufferView
     | NodeJS.ReadableStream
     | string
-    | URLSearchParams;
+    | URLSearchParams
+    | FormData;
```

Comment:
You'll need to import the `FormData` explicitly. See my [earlier comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42364#issuecomment-586269082) for further details.